app/views/layouts/_side-bar.html.erb

```diff
@@ -0,0 +1,36 @@
+<aside class="main-sidebar">
+  <!-- sidebar: style can be found in sidebar.less -->
+  <section class="sidebar">
+    <!-- sidebar menu: : style can be found in sidebar.less -->
+    <ul class="sidebar-menu" data-widget="tree">
+      <li class="header">MAIN NAVIGATION</li>
+      <li class="active treeview">
+        <a href="#">
+          <i class="fa fa-dashboard"></i> <span>Categories</span>
+          <span class="pull-right-container">
+            <i class="fa fa-angle-left pull-right"></i>
+          </span>
+        </a>
+        <ul class="treeview-menu">
+          <li class=""><a href="index.html"><i class="fa fa-circle-o"></i> Dress</a></li>
+          <li><a href="index2.html"><i class="fa fa-circle-o"></i> Suit</a></li>
```

Comment:
Cái nào có url là em cứ để dấu # nhe . Ví dụ href="index.html" thì chuyển thành href="#". Sữa lại hết toàn bộ các href luôn em nhe. Để khi click vào không bị chết trang